templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -608,11 +608,14 @@ function showStartDate( ) {
 }
 
 {/literal}{/if}{literal}
+var thousandMarker = "{/literal}{$config->monetaryThousandSeparator}{literal}";
+var seperator = "{/literal}{$config->monetaryDecimalPoint}{literal}";
```

Comment:
Argh repeat after me sepArator

someone mispelt that in the code base ages ago & now everyone thinks it's the right spelling!